.github/workflows/system-tests-official.yml

```diff
@@ -0,0 +1,129 @@
+name: System Tests - Official
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      # A workaround to trigger the workflow for pull requests from forked repository,
+      # which does not have access to secrets.
+      #
+      # This is also useful for testing the workflow without opening a pull request.
+      - tmp/*
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch: {}
+  schedule:
```

Comment:
Is it necessary to schedule it periodically?

The pull request description mentioned `this new workflow only runs on pushes on master`, however, the code does not matched the description.

Reply:
Done in [59f0358](https://github.com/DataDog/dd-trace-rb/pull/4774/commits/59f0358788a9acc079672376255fe11d38e5ac05)